src/slack/actions/registerTeam.ts

```diff
@@ -64,9 +64,9 @@ function register(bolt: App): void {
 
     const teamName = retrieveViewValuesForField(view, registerTeamViewConstants.fields.teamName, 'plainTextInput') as string;
     const projectDescription = retrieveViewValuesForField(view, registerTeamViewConstants.fields.projectDescription, 'plainTextInput') as string;
-    const tableNumber = retrieveViewValuesForField(view, registerTeamViewConstants.fields.tableNumber, 'plainTextInput') as string;
+    const tableNumber = +retrieveViewValuesForField(view, registerTeamViewConstants.fields.tableNumber, 'plainTextInput');
```

Comment:
Can we use `parseInt()` here? The plus is confusing to those that don't know how Javascript converts things.